app/assets/javascripts/webchat/library.js

```diff
@@ -27,14 +23,19 @@
       if (!availabilityUrl || !openUrl) {
         throw Error.new('urls for webchat not defined')
       }
-      $openButton.on('click', handleOpenChat)
+
+      if (redirectUrl === true) {
```

Comment:
Just curious why this `if` statement wraps the button add listener? I don't see the equivalent in the original code.

Reply:
Ah, I think I was being overly cautious, because the "open button" only exists when `data-redirect` is true. However, you are right it in that it's not needed. I'll swap it for you selection below. Thanks!